model/versioning/VersionedGridFieldDetailForm.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * Extends {@see GridFieldDetailForm}
+ */
+class VersionedGridFieldDetailForm extends Extension {
```

Comment:
Yes, I still believe versioning should be a separate module... but I guess it remains to be seen if that's possible yet.